styles/styles.css

```diff
@@ -242,30 +241,10 @@ main img {
   width: 100%;
 }
 
-/* sections: 56px spacing at the bottom */
 main .section {
   padding-bottom: 56px;
 }
 
-
-/**
- * Styles for embedding video content.
- */
-.video-embed-container {
```

Comment:
Out of curiosity: why have these been removed? Are these now part of the UDex resources?

Reply:
This will be remove by https://github.com/urfuwo/hlx-test/pull/257, was letf over from some testing